### DISCUSSION TITLE
Adding the SDXL version of QRCode monster controlNet (used for Stencil)

### DISCUSSION THREAD

**Discussion by lyonnel4** [Acly/krita-ai-diffusion#690] [Ideas]
Hi.
there is a SDXL version of that model [here](https://huggingface.co/monster-labs/control_v1p_sdxl_qrcode_monster/tree/main).
it would be very nice if you add it to the plugin in.

Thanks a lot for that wonderful plugin

**Comment by ralienpp** [comment]
What model is used when the „stencil” feature of the control layer is invoked?
